src/app/language-support/page.tsx

```diff
@@ -2,24 +2,43 @@
 
 import { getAllInterpretation } from '@/api/supabase/queries/interpretation';
 import { getAllDocuments } from '@/api/supabase/queries/documentTranslation';
+import { getAllCases } from '@/api/supabase/queries/cases';
 import { useEffect, useState } from 'react';
-import { DocumentTranslation, Interpretation } from '@/types/schema';
+import {
+  DocumentTranslation,
+  Interpretation,
+  CaseListing,
+} from '@/types/schema';
 
 export default function Page() {
-  const [docListings, setDocListings] = useState<DocumentTranslation[]>([]);
-  const [intListings, setIntListings] = useState<Interpretation[]>([]);
+  type AllLanguageSupport = DocumentTranslation | Interpretation | CaseListing;
+  const [allLanguageSupport, setAllLanguageSupport] = useState<
+    AllLanguageSupport[]
+  >([]);
+  // const [docListings, setDocListings] = useState<DocumentTranslation[]>([]);
+  // const [intListings, setIntListings] = useState<Interpretation[]>([]);
 
   useEffect(() => {
     const fetchData = async () => {
       try {
-        const docs = await getAllDocuments();
-        const ints = await getAllInterpretation();
-        setDocListings(docs);
-        setIntListings(ints);
+        const [docListings, intListings, casesInterpretationListings] =
+          await Promise.all([
+            getAllDocuments(),
+            getAllInterpretation(),
+            getAllCases(),
+          ]);
+        setAllLanguageSupport([
+          ...docListings,
+          ...intListings,
+          ...casesInterpretationListings.filter(
+            caseInterpretation => caseInterpretation.needs_interpreter === true,
+          ),
+        ]);
```

Comment:
after adding the `upload_date` field, we can sort them here (and have case interpretations be at the top always)

Reply:
done!